go/cmd/dolt/commands/checkout.go

```diff
@@ -118,8 +140,34 @@ func checkoutNewBranch(ctx context.Context, dEnv *env.DoltEnv, newBranch, startP
 	return checkoutBranch(ctx, dEnv, newBranch)
 }
 
-func checkoutTable(ctx context.Context, dEnv *env.DoltEnv, tables []string) errhand.VerboseError {
-	err := actions.CheckoutTables(ctx, dEnv, tables)
+func checkoutNewBranch(ctx context.Context, dEnv *env.DoltEnv, newBranch string, apr *argparser.ArgParseResults) int {
+	startPt := "head"
+	if apr.NArg() == 1 {
+		startPt = apr.Arg(0)
+	}
+
+	verr := createBranchWithStartPt(ctx, dEnv, newBranch, startPt, false)
+
+	if verr != nil {
+		cli.PrintErrln(verr.Verbose())
+		return 1
+	}
+
+	verr = checkoutBranch(ctx, dEnv, newBranch)
+	if verr != nil {
+		cli.PrintErrln(verr.Verbose())
+		return 1
+	}
+	return 0
+}
+
+func checkoutTablesAndDocs(ctx context.Context, dEnv *env.DoltEnv, tables []string, docs []doltdb.DocDetails) errhand.VerboseError {
+	unknown, err := actions.CheckoutTablesAndDocs(ctx, dEnv, tables, docs)
+
+	if unknown != "" {
```

Comment:
Once you make changes in the CheckoutTablesAndDocs method signature, you can eliminate this error handling block (already handled below)

Reply:
Removed error handling block: https://github.com/liquidata-inc/dolt/pull/299/files#diff-7bbaaa4ef160f9fffed44cef87371fa5R43

And now only returning one error param: https://github.com/liquidata-inc/dolt/pull/299/files#diff-7bbaaa4ef160f9fffed44cef87371fa5R53